aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,19 +1,17 @@
 ---
-remote_endpoint: https://fullnode.mainnet.aptoslabs.com
-name: "TBD"
+remote_endpoint: ~
+name: "v1.21.0"
 proposals:
-  - name: proposal_1_upgrade_framework
+  - name: step_1_upgrade_framework
```

Comment:
`proposal_` is a more accurate name here.

Technically `steps` refer to the items under `update_sequence`